homeassistant/components/hue/config_flow.py

```diff
@@ -169,7 +178,13 @@ def _async_get_bridge(self, host: str, bridge_id: Optional[str] = None):
         bridge = self._async_get_bridge(host, discovery_info[ssdp.ATTR_UPNP_SERIAL])
 
         await self.async_set_unique_id(bridge.id)
-        self._abort_if_unique_id_configured()
+        try:
+            self._abort_if_unique_id_configured()
+        except AbortFlow as abort:
```

Comment:
Remove this try except and instead `return self._update_existing_entry(entry)`.

Inside `_update_existing_entry` do `return self.async_abort(...)` at the end.